docs/guide/samples-community-projects.md

```diff
@@ -0,0 +1,15 @@
+## Community Projects
+
+If the robot you are working with is not officially included in webots, take a look at the Community Projects repository. There is a chance, someone already converted it for webots and was nice enough to share for everybody.
+
+<https://github.com/cyberbotics/community-projects>
+
+If you want to add the whole community-projects repository to webots, you can follow these steps:
+
+1. Download or clone the repository to a folder of your choice. 
+
+2. Open webots and go to `Tools > Preferences > Extra Prjocets Path` and add the path to the folder you downloaded. 
+
+3. Add robots by pressing the `Add new object` button (Ctrl + Shit +A) and selecting `PROTO Nodes (Extra Projects)`
+
+4. Be awesome and add things to the Community Projects repository. It doesnt have to be a robot model, it can also be a sample-world or controller.
```

Comment:
```suggestion
4. Be awesome and add things to the Community Projects repository. It doesn't have to be a robot model, it can also be a sample world or controller.
```